src/pages/Home/hooks/useValidateCommit.jsx

```diff
@@ -17,14 +17,15 @@ const useValidateCommit = () => {
   const handleCheckCommitQuality = useCallback(
     async (event) => {
       event.preventDefault();
-      setIsLoading(true);
 
       const formData = new FormData(event.target);
       const repositoryURL = Object.fromEntries(
         formData.entries()
       ).repositoryURL;
 
       try {
+        setIsLoading(true);
+
```

Comment:
```suggestion
      try {
        setIsLoading(true);

        const formData = new FormData(event.target);
        const repositoryURL = Object.fromEntries(
          formData.entries()
        ).repositoryURL;
```

이렇게, repositoryURL 을 만드는 과정에서도 오류가 날 수 있어서 이렇게 모두 try 문에 추가하는 것이 좋을 것 같아요!